src/kakarot/instructions/system_operations.cairo

```diff
@@ -516,6 +517,12 @@ namespace CallHelper {
                 value=call_args.value,
             );
             let (starknet_contract_address) = Accounts.compute_starknet_address(call_args.address);
+            tempvar return_info: model.ReturnInfo* = new model.ReturnInfo(
```

Comment:
ok here i understand,
we init a returninfo struct with ->
len = 0, 
return_data = empty array
size = requested return size
offset = offset in memory to store the data

Reply:
can this declaration live before the if branch? to avoid duplication below?